src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeSummaryRepositoryTest.java

```diff
@@ -0,0 +1,193 @@
+package uk.gov.hmcts.reform.blobrouter.data;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.blobrouter.data.model.EnvelopeSummary;
+import uk.gov.hmcts.reform.blobrouter.data.model.EventRecord;
+import uk.gov.hmcts.reform.blobrouter.data.model.NewEnvelope;
+import uk.gov.hmcts.reform.blobrouter.data.model.NewEventRecord;
+import uk.gov.hmcts.reform.blobrouter.data.model.Status;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.UUID;
+
+import static java.time.ZoneOffset.UTC;
+import static java.util.Comparator.comparingLong;
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Event.DELETED;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Event.DELETED_FROM_REJECTED;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Event.DISPATCHED;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Event.DUPLICATE_REJECTED;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Event.FILE_PROCESSING_STARTED;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Event.REJECTED;
+
+@ActiveProfiles({"integration-test", "db-test"})
+@SpringBootTest
+public class EnvelopeSummaryRepositoryTest {
+    @Autowired private EnvelopeRepository envelopeRepository;
+    @Autowired private EventRecordRepository eventRecordRepository;
+    @Autowired private EnvelopeSummaryRepository repo;
```

Comment:
Given two other repositories have long names, it would be good for this one to be called `envelopeSummaryRepository` for clarity.